pulser/devices/interaction_coefficients/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Pulser Development Team
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""C_6/hbar (in  um^6 / us`), coeffs for Rydberg levels between 50 and 100."""
+
+import json
+from pathlib import PurePath
+
+_json_dict = json.load(open(PurePath(__file__).parent / "C6_coeffs.json"))
+C6_dict = {int(key): value for key, value in _json_dict.items()}
```

Comment:
nit: This is a style thing, we shouldn't name a variable with an upper-case letter (sorry, I just remembered this now). Let's go with `c6_dict` instead

Reply:
Reminder to do this too